packages/fxa-auth-server/lib/senders/emails/templates/passwordForgotOtp/index.mjml

```diff
@@ -0,0 +1,37 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public #
+License, v. 2.0. If a copy of the MPL was not distributed with this # file, You
+can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="password-forgot-otp-title">Your confirmation code</span>
+    </mj-text>
+
+    <mj-text css-class="text-body">
+      <span data-l10n-id="password-forgot-otp-request">
+        We received a request to reset your password. Enter this code to confirm it’s really you.
```

Comment:
I like it, but probably another thing to run by product/copy.

Reply:
I got this from the jira description which I think came from the Figma mocks.  I think there's product input on it already?  @vpomerleau knows more than me prehaps.

Reply:
I think the copy may have been updated.

As far as I know, this is the latest:
![image](https://github.com/mozilla/fxa/assets/22231637/a352e9d2-0a09-4cd6-8d1e-b014abd31d70)

```
Title: Forgot your password?

Body:
We received a request for a password change on your Mozilla account from:

{device info block}

~~If yes, h~~Here is your confirmation code to proceed:

{code}
```

The `If yes,` seems to come from another template and doesn't fit in this context.

Reply:
I think the `If yes,` works in the context of the previous heading of `Forgot your password?` (although the context gets a bit lost with the "We received a request..." and the browser info and timestamps.)

Not sure if it's easier to reorder to:

```
Forgot your password?

If yes, here is your confirmation code to proceed...

If you don't take this action, change your password right away.

We received a request for a change on your Mozilla account from: ...

For more info, visit Mozilla Support...
```